Components/RoundButton.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import { WithTheme, TouchableRipple } from 'react-native-paper';
+import { TouchableRipple } from 'react-native-paper';
 
 export default function RoundedButton (props) {
```

Comment:
Not a big deal, but just so you know other styles of code, most people typically do:

```
export default ({onPress, text}) => (
    // Your component
)